troposphere/asg-elb-dns.py

```diff
@@ -1,14 +1,26 @@
 from troposphere import Join, Output, GetAtt
 from troposphere import Parameter, Ref, Template
-from troposphere.autoscaling import AutoScalingGroup, Tag
-from troposphere.policies import UpdatePolicy, AutoScalingRollingUpdate, CreationPolicy, AutoScalingCreationPolicy, ResourceSignal
+from troposphere.autoscaling import AutoScalingGroup, Tag, ScalingPolicy
+from troposphere.cloudwatch import Alarm, MetricDimension
+from troposphere.policies import UpdatePolicy, AutoScalingRollingUpdate, CreationPolicy, \
+    AutoScalingCreationPolicy, ResourceSignal
 import troposphere.elasticloadbalancing as elb
 from troposphere.route53 import RecordSetType
+import sys
 
 __author__ = 'Jose Armesto'
 
 
 def generate_cloudformation_template():
+    enable_elb = sys.argv[1]
+    input_scaling_policies = eval(sys.argv[2])
+    input_alarms = eval(sys.argv[3])
+
+    if enable_elb == 'True':
```

Comment:
```python
enable_elb = enable_elb == 'True' # or
enable_elb = bool(enable_elb)
```

I understand that you might not want to depend on 3rd party libraries, but I'd use [Click](http://click.pocoo.org/5/) for everything that is related with argument parsing.